### PR TITLE
chore: In rpc.proto and getGrpcError, add VM error codes

### PR DESCRIPTION
**Pull request by jefft0**
This PR improves error handling.
* In rpc.proto, improve the wording of some API functions
* In rpc.proto `ErrCode`, add VM errors from https://github.com/gnolang/gno/blob/master/gno.land/pkg/sdk/vm/errors.go
* Add these errors to api.go `getGrpcError` 
* Run `make regenerate`

### CODE REVIEW

File: api/gen/es/rpc_connect.d.ts [hunk 1]
```diff
@@ -173,7 +173,9 @@ export declare const GnoNativeService: {
       readonly kind: MethodKind.Unary,
     },
     /**
-     * SelectAccount selects the active account to use for later operations
+     * SelectAccount selects the active account to use for later operations. If the response has_password is
+     * false, then you should set the password before using a method which needs it.
+     * If the key doesn't exist, return [ErrCode](#land.gno.gnomobile.v1.ErrCode).ErrCryptoKeyNotFound.
```
Comment (D4ryl00) [inline api/gen/es/rpc_connect.d.ts:178]: Why `gnomobile` and not `gnonative`?

File: api/gen/es/rpc_connect.d.ts [hunk 2]
```diff
@@ -239,6 +241,8 @@ export declare const GnoNativeService: {
     },
     /**
      * Make an ABCI query to the remote node.
+     * If the request path is unrecognized, return [ErrCode](#land.gno.gnomobile.v1.ErrCode).ErrUnknownRequest.
```
Comment (D4ryl00) [inline api/gen/es/rpc_connect.d.ts:244]: Why gnomobile and not gnonative?

File: api/gen/es/rpc_connect.d.ts [hunk 3]
```diff
@@ -239,6 +241,8 @@ export declare const GnoNativeService: {
...
+     * If the request data has a package path that is unrecognized, return [ErrCode](#land.gno.gnomobile.v1.ErrCode).ErrInvalidPkgPath.
```
Comment (D4ryl00) [inline api/gen/es/rpc_connect.d.ts:245]: Why gnomobile and not gnonative?

**Review by D4ryl00 — APPROVED** [review]
LGTM, see the 3 same comments please.

File: api/gen/es/rpc_connect.d.ts (hunk 1 above)
Comment (jefft0) [reply api/gen/es/rpc_connect.d.ts]: Thanks for catching this. I committed a fix. (I started working on the PR a long time ago ;-)

File: api/gen/es/rpc_connect.d.ts (hunk 2 above)
Comment (jefft0) [reply api/gen/es/rpc_connect.d.ts]: Thanks for catching this. I committed a fix. (I started working on the PR a long time ago ;-)

File: api/gen/es/rpc_connect.d.ts (hunk 3 above)
Comment (jefft0) [reply api/gen/es/rpc_connect.d.ts]: Thanks for catching this. I committed a fix. (I started working on the PR a long time ago ;-)

**Pull request merged**